Monika After Story/game/script-ch30.rpy

```diff
@@ -816,10 +813,12 @@ label spaceroom(start_bg=None, hide_mask=None, hide_monika=False, dissolve_all=F
 
         day_mode = mas_current_background.isFltDay()
 
-    if scene_change:
-        scene black
```

Comment:
the `scene` statement is an easy way to remove all displayables quickly. Much better than having explicit hide statements like the ones you ended up adding elsewhere. If a scene change occurs, we only need to consider what we want to show, not what needs to be hidden.

Reply:
Works perfectly now with #7036.